### ISSUE TITLE
error installing Jupyter

### ISSUE THREAD

**Issue by hilacezana** [mrdbourke/zero-to-mastery-ml#70]
hi, I get this error while trying to install Jupyter through terminal in macOS.
how can I fix it?

thanks

<img width="770" alt="Screenshot 2023-12-04 at 11 22 13" src="https://github.com/mrdbourke/zero-to-mastery-ml/assets/152856978/55f09028-973e-43d4-8ca4-c6bc3ac4697b">

**Comment by Sohaib90** [comment]
Can you try downgrading your python version ? The package requires python >= 2.7, <2.8.0a0

**Comment by vfonsecal** [comment]
Hello I have the same issue.

Installing an older version of python does not solve the problem, as the Miniconda version on the the course video asked  to be installed contains Python 3.7. Can anyone help?

**Comment by GhHiva** [comment]
I have the same problem 🫠

**Comment by Pratikbhai19392** [comment]
Yes, I Also Have the same problem with how I can install Jupyter I do not know. Every Permutation and combination is done but still the same problem anyone help?

**Comment by kRiShNa-429407** [comment]
Try to install via pip

**Comment by Shuraimi** [comment]
Hello 👋,
Even I had the same problem. This is just a problem of incompatibility between these different libraries such as Numpy, pandas Matplotlib Scikit Learn and jupyter.

When creating an environment, install the libraries for python 3.10/3.11 as jupyter notebook doesn't support python 3.12.

`conda install python=3.11 numpy pandas matplotlib scikit-learn jupyter`

If this doesn't help replace 3.11 with 3.10.

Using this command, all packages will be downloaded with python 3.11/3.10 which is compatible with all these libraries.
